pkg/cli/admin/etcd/dnshelpers.go

```diff
@@ -0,0 +1,119 @@
+package etcd
```

Comment:
maybe it's time to consider these helpers in library-go?

Reply:
Maybe, or as a pre-req for this merging?